frontend/src/components/Contributions/ContributionListInfo.vue

```diff
@@ -0,0 +1,53 @@
+<template>
```

Comment:
i think the naming of this is not optimal. This is the Alert on top of the App, correct?
Consider to name it `GradidoNotification` or `UserNotification` or similar - or is it really only on contribution list?

Reply:
done, name is **GradidoNotification**